_posts/blog/2019-08-01-lab-meeting.md

```diff
@@ -0,0 +1,58 @@
+Because she’s used to sharing knowledge and making things clear :green_heart:, it can be heard to work out the right level     of explanation for other experts. 
+  * Tip for writing - start with bullet points and then build up a path of what else needs explaining.
+
+Georgia: wants to know how to help people help each other in the lab meetings. 
+  * Sarah suggested that since she doesn’t have the domain knowledge to help everyone, if we talked about technical challenges, she would be better able to help.
+  * Georgia suggested having some kind of shared goal which we can all work together on. 
+  * Isla suggested being explicit and bringing an extremely practical problem everyone could on.
+  * Sarah  suggested something like a coding club - maybe there could be a small problem but one entirely contained within the meeting? Something like REG tech talks. 
+
+Sarah: Wants tips on working on a project if the content is overwhelming (a.k.a anyone familiar with Guassian Processes? :sweat_smile:)
+
+Kirstie has been thinking a lot about the need for trust in communities – specifically, that we need to trust each other to be able to ask each other questions. 
+She’s been thinking about what the barriers to trust are in the group are. She’d love to know what has worked in the past to build a supportive team and environment.
+  * Georgia: Overall, it feels like there is a lot of trust at the Turing - people do help each other out.
+  * Georgia: if shyness and lack of self-confidence is an issue for contributing to a group, presenting, asking questions etc., is exposure therapy productive or counterproductive for overcoming it? 
+    * Sarah: you have to strike a balance, e.g. finding a friendly audience to start with (but not avoiding it entirely)
+  * Georgia asked Sarah what makes the Binder community supportive?
+        * EMOJIS!!! :sparkles: :sparkling_heart: :smiley:
+        * Friendly messages
+        * Never miss an opportunity to remind people they're doing a good job - very important in the Binder community because they're so distributed.
+
+Georgia would like ideas about the best ways of learning physics?
+  * Sarah suggests looking at science videos on YouTube! There are also fun experiments that you can do yourself at home, for example experimenting with static electricity using balloons - stay away from dry manuals
+  * Check out: https://www.youtube.com/results?search_query=kurzgesagt 
+	* Find someone who’s a physicist and ask them to explain directly
+  * Isla gave this awesome (wise) advice: You start learning about physics when you're a baby when you pick things up and drop them to see what happens – carry on with this attitude. Stay curious! :baby::woman::older_woman:
+  * Georgia noticed that a lot of popular science books explain things in terms of metaphors, but do the metaphors capture everything? (Consensus in the room: probably not?)
```

Comment:
I suspect you can drop this unless you want to expand it. As it is it's a little bullet point-y.